js/form.js

```diff
@@ -22,10 +22,26 @@ capacity.addEventListener('change',() => {
 });
 
 typeHousing.addEventListener('change', () => {
-  for( const key in MINIMUMCOSTHOUSING){
+  for(const key in MINIMUMCOSTHOUSING){
```

Comment:
```suggestion
  for (const key in MINIMUMCOSTHOUSING) {
```
Тогда ещё можно добавить пробелы здесь.